File: Curso Logica de Programação Neri Neitzke/Algoritmos em Java/media_para.java
// Prof Neri Aldoir Neitzke
// www.INFORMATICON.com.br - videoaulasneri@gmail.com
//Problema:Ler a nota de 10 alunos, calcular a media e mostrar essa média. Só aceitar numeros de 0 a 10
import javax.swing.JOptionPane;
class media_para
{
   public static void main(String args[])
   {
      float nota,calc_media=0, soma_notas=0; //criei as variáveis
      int i;
      for(i=1;i<=10;i++)
      {
         nota = Float.parseFloat(JOptionPane.showInputDialog("Digite a "+i+". nota de 10 notas. Somente notas de 0 a 10"));
         if ((nota >=0) && (nota <= 10))
            soma_notas = soma_notas + nota;
         else
            {
                JOptionPane.showMessageDialog(null,"Nota Invalida, digite apenas notas de  0 a 10");
                i=i-1;
             }
      }
      calc_media = soma_notas / 10;
      JOptionPane.showMessageDialog(null,"A media geral das notas .: "+calc_media);
   }
}

File: Curso Logica de Programação Neri Neitzke/Algoritmos em Pascal/pascal/media_para.pas
// Prof Neri Aldoir Neitzke
// www.INFORMATICON.com.br - videoaulasneri@gmail.com
//Problema:Ler a nota de 10 alunos, calcular a media e mostrar essa média

program media_para;
var
   nota,calc_media, soma_notas: real;  //definição variáveis
   i:integer;
begin
   soma_notas :=0;
   for i:=1 to 10 do
   begin
       write('Insira a ',i,'. de 10 notas.: ');
       readln(nota);
       Soma_notas :=soma_notas + nota;
   end;
   calc_media := soma_notas / 10;
   writeln('A media geral dos 10 alunos .: ',calc_media:6:2);  //saída
   write('...................Final do Sistema..................... ');
   readln();
end.
